packages/labs/src/tests/frameworks/react/create-component_test.ts

```diff
@@ -89,7 +89,7 @@ suite('React createComponent', () => {
     props?: ReactModule.ComponentProps<typeof BasicElementComponent>
   ) => {
     window.ReactDOM.render(
-      window.React.createElement(BasicElementComponent, props),
+      window.React.createElement(BasicElementComponent, props, props?.children),
```

Comment:
Is there any harm in duplicating `children` in `props` and in the 3rd arg?

Reply:
It looks like you're supposed to pass children separate from props but it looks harmless to include children in props, see this [example](https://babeljs.io/en/repl#?browsers=defaults%2C%20not%20ie%2011%2C%20not%20ie_mob%2011&build=&builtIns=false&spec=false&loose=false&code_lz=MYGwhgzhAEBiD29oFMAeAXZA7AJjASsmMOgHQDC8AtgA7xbbrQDeAUNNAE7Y7KcAUAShbsOXZOgCunLNAA8OAJYA3AHzN0AC0URSNTvBq7g2kDm5YAvnID0StQG5Rl1i9ahIMABLIQIJGiYuAREJBTUdAxYTGwcFrwCwrFi3FIy8ghIJopmFgC8Gtq6-obGpubYlupaOnoGRqTZuZUKKqq29u02mapOHC5uhMToACIA8gCypPF8_KJyPn5Z4FAAcmBUyHkARABmiNAARmCc2-2d2h1ttov-qgA0ojjwwJKb0aQA5hIAoiDI73QACEAJ4ASRw_AA5AZ4OgoYJWIIHEA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=true&presets=react&prettier=false&targets=&version=7.12.10&externalPlugins=).

Reply:
It looks fine to remove so will do so.